src/commonMain/kotlin/org/kbson/BsonSymbol.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kbson
+
+/**
+ * A representation of the BSON Symbol type.
+ *
+ * <p>Note: It's deprecated in BSON Specification and present here only for compatibility reasons.
+ */
+class BsonSymbol(val symbol: String) : BsonValue() {
```

Comment:
Not sure about naming conventions, but most of the other types use `value` for this argument. Not sure if that is something we want to use across all types or make them more descriptive like this one. In which case e.g. `BsonDateTime` should probably rename its constructor argument to something more readable.

Reply:
I've changed it back to value - just to match org.bson - not sure why I had changed it.